templates/catalog/_partials/products-top.tpl

```diff
@@ -4,15 +4,15 @@
  *}
 <div id="js-product-list-top" class="products-selection">
   <div class="products-selections-filters row">
-    <div class="col-12 col-md-6 total-products order-2 order-md-1 mt-4 mt-md-0">
+    <div class="col-12 col-md-12 col-lg-6 total-products order-2 order-lg-1 mt-4 mt-lg-0">
```

Comment:
@JBLach Hi Jakub, can we please simplify this? `col-12 col-md-12 col-lg-6` can be just `col-lg-6`

The same please on line 15 `col-md-12 col-lg-6` -> `col-lg-6`.

Otherwise, I think it's good. Maybe just there will be a better way to specify the modal width. I would just say that modal will be `100% - some padding`, up to certain max width. WDYT @NeOMakinG @mparvazi

Reply:
Hi, I followed the shopingcart modal example, maybe we could create a helper class for modals below the 992px viewport so we don't have to overwrite every modal that has too much content, something similar to BS class `modal-fullscreen-lg-down` but only with maximum width with padding.  WDYT @Hlavtox @NeOMakinG @mparvazi

Reply:
> maybe we could create a helper class for modals below the 992px viewport so we don't have to overwrite every modal that has too much content, something similar to BS class modal-fullscreen-lg-down but only with maximum width with padding.

Yes, that would be good!

Change this also please - `col-12 col-lg-6` to `col-lg-6`

Reply:
> Hi, I followed the shopingcart modal example, maybe we could create a helper class for modals below the 992px viewport so we don't have to overwrite every modal that has too much content, something similar to BS class `modal-fullscreen-lg-down` but only with maximum width with padding. WDYT @Hlavtox @NeOMakinG @mparvazi

If we have multiple modals with the same behavior, a helper class could be better than duplicating media queries of course!